katportalclient/test/test_client.py

```diff
@@ -588,8 +588,7 @@ def test_sensor_names_single_sensor_valid(self):
         sensor_name_filter = 'anc_weather_wind_speed'
 
         self.mock_http_async_client().fetch.side_effect = mock_async_fetcher(
-            valid_response='[{}]'.format(
-                sensor_json['anc_weather_wind_speed']),
+            valid_response=('"data": [%s]'%sensor_json['anc_weather_wind_speed']),
```

Comment:
Should be a space either side of the `%` operator. 

Also, is this response still valid JSON, and equivalent to the response we'd get from katstore?   I notice `test_sensor_detail` uses enclosing braces `{ ... }`.